### PR TITLE
partners contact us

### PR DESCRIPTION
**Pull request by nerik**
screen shot 2016-08-29 at 11 06 10

### CODE REVIEW

File: app/src/components/Partners.jsx [hunk 1]
```diff
@@ -23,11 +23,12 @@ class Partners extends Component {
         attribution="US Coast Guard"
         showLDFLogo
       />
-      <div className={StaticPageStyles['l-static-page']}>
```
Comment (andresgnlez) [inline app/src/components/Partners.jsx]: Why did you remove the layout?

Comment (nerik) [inline app/src/components/Partners.jsx]: The margin-bottom added a white stripe because the last section is now blue, which looked rid. So I needed to remove the margin-bottom, so eventually moving the margin-top to the first section instead of the whole page made more sense.
